crates/storage/src/db.rs

```diff
@@ -191,3 +196,29 @@ impl ReamDB {
         }
     }
 }
+
+pub fn reset_db(path: Option<PathBuf>) -> anyhow::Result<()> {
+    let db_path = path.clone().ok_or_else(|| anyhow!("Path does not exist"))?;
+
+    if db_path.exists() {
+        println!(
+            "Are you sure you want to remove the database at {:?}? (y/n):",
+            path
+        );
+        io::stdout().flush().unwrap();
+
+        let mut input = String::new();
+        io::stdin().read_line(&mut input).unwrap();
+        if input.trim().eq_ignore_ascii_case("y") {
+            match fs::remove_dir_all(&db_path) {
+                Ok(_) => println!("Database cleared successfully."),
+                Err(e) => eprintln!("Failed to remove database: {}", e),
```

Comment:
```suggestion
                Err(e) => eprintln!("Failed to remove database: {}", e),
```

shouldn't we return an error if an error happens